cdap-app-fabric-tests/src/test/java/co/cask/cdap/internal/app/services/http/handlers/WorkflowStatsSLAHttpHandlerTest.java

```diff
@@ -63,13 +65,30 @@
   private static MetricStore metricStore;
   private static Store store;
 
+  private int sourceId;
+
   @BeforeClass
   public static void beforeClass() throws Throwable {
     AppFabricTestBase.beforeClass();
     store = getInjector().getInstance(DefaultStore.class);
     metricStore = getInjector().getInstance(MetricStore.class);
   }
 
+  private void setStartAndRunning(final ProgramId id, final String pid, final long startTime) {
+    setStartAndRunning(id, pid, startTime, ImmutableMap.<String, String>of(),
+                       ImmutableMap.<String, String>of());
+
+  }
+
+
+  private void setStartAndRunning(final ProgramId id, final String pid, final long startTime,
+                                  final Map<String, String> runtimeArgs,
+                                  final Map<String, String> systemArgs) {
+    store.setStart(id, pid, startTime, null, runtimeArgs, systemArgs,
+                   ByteBuffer.allocate(++sourceId).array());
+    store.setRunning(id, pid, startTime + 1, null, ByteBuffer.allocate(++sourceId).array());
```

Comment:
Similar to the other two methods, if you place this as a static method in a class that all tests can access, you don't need to copy it to every test case.